app/src/main/res/layout/activity_chat.xml

```diff
@@ -55,5 +54,17 @@
             app:layout_constraintBottom_toBottomOf="parent"
             app:layout_constraintEnd_toEndOf="parent"
             app:layout_constraintVertical_bias="0.5" />
+
+        <ImageButton
+            android:id="@+id/fileButton"
+            android:layout_width="@dimen/chat_square_send_button"
+            android:layout_height="@dimen/chat_square_send_button"
+            android:layout_marginEnd="@dimen/meshim_margin_small"
+            android:onClick="alertDialogToChooseFile"
```

Comment:
For our other button we set the action in code instead of here. It would be good to change either this button or that one to be consistent.